npc/quests/first_class/tu_sword.txt

```diff
@@ -1585,8 +1585,7 @@ geffen,154,143,3	script	Dequ'ee	734,{
 				mes "the third code";
 				mes "would be...?";
 				next;
-				switch(select("Muetro:Bankley")) {
-				case 1:
+				if(select("Muetro:Bankley") == 1) {:
```

Comment:
```suggestion
				if(select("Muetro:Bankley") == 1) {
```
Looks like a leftover from the switch.

```
 Error: [Error]: 
script error on npc/quests/first_class/tu_sword.txt line 1588
    script:add_word: invalid word. A word consists of undercores and/or alphanumeric characters, and valid variable prefixes/postfixes.
  1583 : 				mes "Alright, then";
  1584 : 				mes "the person with";
  1585 : 				mes "the third code";
  1586 : 				mes "would be...?";
  1587 : 				next;
* 1588 : 				if(select("Muetro:Bankley") == 1) {':'
  1589 : 					mes "[Dequ'ee]";
  1590 : 					mes "Geil, Hans,";
  1591 : 					mes "Muetro and Bankley.";
  1592 : 					mes "So then the full";
  1593 : 					mes "code would be...";
```

Reply:
Yeah I know, but I'm rewriting that part because is pretty much the same code multiple times, so that will be gone anyways